libclamav/matcher-ac.c

```diff
@@ -641,10 +641,35 @@ static void ac_free_special(struct cli_ac_patt *p)
     MPOOL_FREE(mempool, p->special_table);
 }
 
+/*Find all duplicate pointers and zero them so that we won't get double-free
+ * errors when we iterate over the cli_matcher to free all the trans
+ * pointers.*/
+static void zero_duplicate_trans_nodes(struct cli_matcher *root, const size_t idx) {
+    size_t j;
+    for (j = 0; j < root->ac_nodes; j++) {
```

Comment:
Inconsistent spacing.  Surrounding style adds a blank line after declarations, and no blank line before the function's closing brace.